packages/admin-app/src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert, AlertProps } from '@mui/material';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+export interface ErrorMessageProps extends AlertProps {
+    MessageId: string;
+}
+
+export const ErrorMessage: React.FC<ErrorMessageProps> = (props) => {
+    const { MessageId } = props;
+    const alertProps = props as AlertProps;
+
+    return (
+        // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
This eslint disabling is used 4 times throughout the project, should we maybe just disable it globally?

Reply:
So we are disabling it everytime we use the spreading?  If that is the case, I would agree to disable it.

Reply:
It's actually a little more nuanced than that.  It seems to be necessary and disabled in components that extend another component and need to enable consumers to use the original component's props plus an extra property or two.  In contexts outside of that scenario it's maybe still a good idea to disable prop spreading.  Unfortunately, I don't have enough experience yet to know how common that lint rule might prove useful.